.github/workflows/lint.yml

```
@@ -0,0 +1,25 @@
+        uses: actions/setup-node@v4
+        with:
+          node-version: 18
+
+      - name: Install dependencies
+        run: npm ci
+
+      - name: Run ESLint
+        run: npm run lint
+
+      - name: Run Prettier check
+        run: npx prettier --check . 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)